packages/domain-events/src/Domain/Event/UserChangedEmailEventPayload.ts

```diff
@@ -0,0 +1,5 @@
+export interface UserChangedEmailEventPayload {
```

Comment:
Nitpick but the pattern with other events is that the verb is last. So this would be `UserEmailChanged` event.

Reply:
Fixed in: https://github.com/standardnotes/snjs/pull/409